drape/color.hpp

```diff
@@ -6,63 +6,70 @@
 #include <sstream>
 #include <string>
 
+#define EXTRACT_BYTE(x, n) (static_cast<uint8_t>(((x) >> 8 * (n)) & 0xFF))
+#define CHANNEL_TO_FLOAT(x) ((x) / 255.0 /* MaxChannelValue */)
+
 namespace dp
 {
 struct Color
 {
-  Color();
-  explicit Color(uint32_t rgba);
-  Color(uint8_t red, uint8_t green, uint8_t blue, uint8_t alpha);
+  constexpr Color() : m_rgba(255 /* MaxChannelValue */) {}
```

Comment:
Write a comment, what default color actually is. What is "MaxChannelValue"? Is it (0, 0, 0, alfa = 255) ?